### PR TITLE
S04 us35 02 edit admin form course inscription

### PR DESCRIPTION
**Pull request by ChristianOlmedo**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]

File: src/View/Admin/Includes/_show_courses_inscription.html.twig [hunk 1]
```diff
@@ -10,6 +10,7 @@
                 <th scope="col">Nom d'un parent</th>
                 <th scope="col">Adresse email</th>
                 <th scope="col">Numéro de téléphone</th>
+                <th></th>
```
Comment (LucyMerlier) [inline src/View/Admin/Includes/_show_courses_inscription.html.twig:13]: why empty th tag?

File: src/View/User/Includes/_inscription_form_course.html.twig [hunk 2]
```diff
@@ -83,9 +83,9 @@
                 <div class="mb-3 field">
                     <label for="phone" class="form-label">
                         <span class="text-danger">*</span>Téléphone :</label>
-                    <input type="text" class="form-control" id="phone" name="phone" placeholder="06.00.00.00.00." required value="{{ course.phone }}" aria-required="true">
+                    <input type="text" class="form-control" id="phone" name="phone" placeholder="06.00.00.00.00." required value="{{ course.phone_number }}" aria-required="true">
```
Comment (LucyMerlier) [inline src/View/User/Includes/_inscription_form_course.html.twig:86]: there's a type="tel" in html, that is basically a type="text" but that opens an appropriate "only numbers" keyboard for phones and tablets ;)

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/Admin/Includes/_show_courses_inscription.html.twig (hunk 1 above)
Comment (ChristianOlmedo) [reply src/View/Admin/Includes/_show_courses_inscription.html.twig:13]: It's due to the fact that i added buttons to delete and edit at the and of the table, thoose th tags allow to enhance esthetic of the table by adding a black line above buttons

File: src/View/User/Includes/_inscription_form_course.html.twig (hunk 2 above)
Comment (ChristianOlmedo) [reply src/View/User/Includes/_inscription_form_course.html.twig:86]: type="tel" seems to return strings too, change text to tel will not change anything

File: src/View/Admin/Includes/_show_courses_inscription.html.twig (hunk 1 above)
Comment (LucyMerlier) [reply src/View/Admin/Includes/_show_courses_inscription.html.twig:13]: why not ^_^

File: src/View/User/Includes/_inscription_form_course.html.twig (hunk 2 above)
Comment (LucyMerlier) [reply src/View/User/Includes/_inscription_form_course.html.twig:86]: as explained above, it does change the UX for phone users, but ok

**Review by LucyMerlier — DISMISSED** [review]

**Pull request merged**